Makefile

```diff
@@ -33,14 +46,25 @@ clean: test-pre-clean
 	rm -f $(DISTNAME).tar.gz $(DISTNAME).zip
 	rm VERSION-FILE
 
-install:
-	mkdir -p $(INSTALL_DIR)
-	install --mode=755 todo.sh $(INSTALL_DIR)
-	mkdir -p $(BASH_COMPLETION_DIR)
-	install --mode=644 todo_completion $(BASH_COMPLETION_DIR)/todo
-	mkdir -p $(CONFIG_DIR)/todo
-	[ -e $(CONFIG_DIR)/todo/config ] || \
-		sed "s/^\(export[ \t]*TODO_DIR=\).*/\1~\/.todo/" todo.cfg > $(CONFIG_DIR)/todo/config
+install: installdirs
+	$(INSTALL_PROGRAM) todo.sh $(DESTDIR)$(bindir)/todo.sh
+	$(INSTALL_DATA) todo_completion $(DESTDIR)$(datarootdir)/bash_completion.d/todo
+	[ -e $(DESTDIR)$(sysconfdir)/todo/config ] || \
+	    sed "s/^\(export[ \t]*TODO_DIR=\).*/\1~\/.todo/" todo.cfg > $(DESTDIR)/$(sysconfdir)/todo/config
+
+uninstall:
+	rm -f $(DESTDIR)$(bindir)/todo.sh
+	rm -f $(DESTDIR)$(datarootdir)/bash_completion.d/todo
+	rm -f $(DESTDIR)/$(sysconfdir)/todo/config
+
+	rmdir $(DESTDIR)$(datarootdir)/bash_completion.d
+	rmdir $(DESTDIR)/$(sysconfdir)/todo
+
+installdirs:
+	mkdir -p $(DESTDIR)$(bindir) \
+	         $(DESTDIR)$(sysconfdir)/todo \
```

Comment:
Why is `$(sysconfdir)` prepending with a slash (`/`) in some cases and not in others.

With slash:
[Line 53](https://github.com/todotxt/todo.txt-cli/pull/239/files#diff-b67911656ef5d18c4ae36cb6741b7965R53)
[Line 58](https://github.com/todotxt/todo.txt-cli/pull/239/files#diff-b67911656ef5d18c4ae36cb6741b7965R58)
[Line 61](https://github.com/todotxt/todo.txt-cli/pull/239/files#diff-b67911656ef5d18c4ae36cb6741b7965R61)

Without slash:

[Line 52](https://github.com/todotxt/todo.txt-cli/pull/239/files#diff-b67911656ef5d18c4ae36cb6741b7965R52)
[Line 65](https://github.com/todotxt/todo.txt-cli/pull/239/files#diff-b67911656ef5d18c4ae36cb6741b7965R65)